starter-code/src/App.jsx

```diff
@@ -0,0 +1,21 @@
+// LE FEU : taper rafc / rfc etc etc pour init easy !
+import React from 'react';
+import Navbar from "./components/Navbar";
+import Main from "./components/Main";
+import Button from "./components/Button";
+import Footer from "./components/Footer";
+
+export default function App() {
+    return (
+        <div>
+            <div className="blue-box">
+                <Navbar />
+                <Main />
+                <Button />
+            </div>
```

Comment:
If you would like to go deeper into making things into a component, you could create a "component" out of the wrapper div that is made for styling purposes.
Eg : 
```
export default function App() {
  return (
    <div>
      <StyledDiv boxColor="blue-box">
        <Navbar />
        <Main />
        <Button />
      </StyledDiv>
      <StyledDiv boxColor="white-box">
        <Footer />
      </StyledDiv>
    </div>
  );
}
```

StyledDiv ----- 
```
import React from "react";

const StyledDiv = props => {
  return <div className={props.boxColor}>{props.children}</div>;
};

export default StyledDiv;

```